aspire_cloud_hosting_policy.md

```diff
@@ -0,0 +1,53 @@
+# Aspire Cloud Hosting Policy
+
+## Preamble
+
+The goal of Aspire Cloud Repository is to provide a robust distributed source for downloading software for extending WordPress and related compatible CMSs. The Distributed Software Repository project is released freely in order to establish a network of trusted repositories which democratize software distribution free from the control of any one person or organization.
+
+This DRAFT policy governs what software will be accepted to host on AspirePress Cloud. At the project's inception, we are providing a general list of requirements and deferring to the corresponding [guidelines available at wordpress.org](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/) for interpretation until they have been fully rewritten in this document. While this document is in draft form, in case of conflicts or updates, the guideline explanations below will supersede any others.
+
+If you believe you have a case where a software package would not be accepted for hosting on wordpress.org, but believe it should be allowed on Aspire Cloud, please contact support@aspirepress.org.
+
+While this Preamble should help understand and interpret the Hosting Policy, it is not a formal part of Policy.
+
+-----
+
+## Aspire Cloud Guidelines, Terms, & Conditions
+
+### Licensing & Other Requirements
+
+1. Plugins must be compatible with the GNU General Public License v2, but may use any license which qualifies.
+2. By submitting your software and updates, you agree to indemnify and hold harmless AspirePress and its network partners for any actions resulting from your software.
+3. Code must be (mostly) human readable.
+4. Developers are responsible for the contents and actions of their plugins.
+5. A stable version of a plugin must be available from its WordPress Plugin Directory page.
```

Comment:
I move to strike this entire clause.  It disqualifies ACF, Gravity Forms, and our own AspireUpdate plugin for one, and contradicts several of our own press releases, including ones mentioning that we're supporting first-party publishing at launch.  Our entire purpose is to make the .org repo ultimately irrelevant.

Possibly we should split this doc into guidelines for plugins/themes already on .org and those we are publishing first-party.  We don't really even need guidelines for the plugins on .org, since they have their own guidelines and we're already slurping up and serving those en masse.  I do suppose we should have the redundant guidelines in place to disqualify plugins such as SCF though 😄 

Also note that many references to plugins throughout this doc may have to be updated to include themes.  Maybe we can define a more generic term such as "extensions", though I have a hard time seeing it expanding beyond plugins and themes (block patterns maybe, but I'd just call those a theme subset)